cli/filelister.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include <cstddef>
 #include <cstring>
+#include <limits.h>
```

Comment:
Please use the C++ include `<climits>` and add a comment to it so it might not be accidentally removed in the future.

Reply:
Thanks for your advices, btw I've close this pr and moved to the main branch [here](https://github.com/danmar/cppcheck/pull/3098).